src/db/db.ts

```diff
@@ -0,0 +1,26 @@
+
+import { FastifyInstance } from 'fastify';
+import fastifyPlugin from 'fastify-plugin';
+import { Client } from 'pg';
+
+const client = new Client({
```

Comment:
can you add here a todo for me to research how to put this config without it being on the git? Not like it has any sensitive info, but I think if it doesn't take much I should change it to be more secure :)

Reply:
I saw places that use ENV variables, so it will be one way, I will add  TODO